### DISCUSSION TITLE
What is the recommended way to reference 'home-brew' Prism assemblies ?

### DISCUSSION THREAD

**Discussion by sigmarsson** [PrismLibrary/Prism#2685] [General] [answered]
If any1 intends to compile Prism , how to reference the assemblies in the project ? It has also the dependency `DryIoc.dll`, which - if as Nuget pkg added - results in this error: 

![image](https://user-images.githubusercontent.com/43340959/166956036-8f64f84a-3f32-4be5-a77b-945629ad22f8.png)

This way apparently not working;

![image](https://user-images.githubusercontent.com/43340959/166956443-a9c28eec-8e89-4583-9e73-3298cad18ba8.png)

What do you advise ? Thanks.

**Comment by brianlagunas** [comment, marked as answer]
You're using the wrong version of DryIoC. If you are compiling Prism source, either update the dependencies and fix any errors from the upgrade or downgrade your DryIoC nuget to the proper version of your Prism build.

**Reply by sigmarsson** [reply]
Thanks.

**Comment by dansiegel** [comment]
You've updated the major version of DryIoc. As you are seeing from the error this contained breaking API changes which Prism was not compiled against. vNext will be compiled against the newer version of DryIoc but in the mean time you will need to back the version of DryIoc down to the lastest 4.x or simply let it resolve transitively through Prism.DryIoc.Uno

**Reply by sigmarsson** [reply]
Thanks. I think I found a bug in github. It is not possible to mark multiple answers as `Mark as answer`
